.github/workflows/codecoverage.yaml

```
@@ -26,7 +26,7 @@ jobs:
           distribution: 'temurin'
 
       - name: Setup Gradle
-        uses: gradle/gradle-build-action@a8f75513eafdebd8141bd1cd4e30fcd194af8dfa # v2
+        uses: gradle/actions/setup-gradle@v3
```

You can grab the hash manually from the tag, one less PR to manage.

---

this will be the exact line if Renovate pins it back:

```suggestion
        uses: gradle/actions/setup-gradle@aff52e5be96935327d77c5529075184377dc4371 # v3
```

---

Proof https://github.com/MythicDrops/mythicdrops-gradle-plugin/commit/3a6fcf9443b75af435b1fa3c211c2d6cc8145f51#diff-0d634959c8276ae976e39ba475e63d0b9ec27824470c79f1971f58fe37c46325R50

---

Just FYI that hash is for 3.0.0-rc.2.

The PR Renovate opened uses the correct hash for v3 final.

---

haha :D, yeah I got confused by the GitHub UI:
https://github.com/gradle/actions/issues/10